api/docs/tough-cookie.permutepath.md

```diff
@@ -0,0 +1,61 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [tough-cookie](./tough-cookie.md) &gt; [permutePath](./tough-cookie.permutepath.md)
+
+## permutePath() function
+
+Generates the permutation of all possible values that [pathMatch()](./tough-cookie.pathmatch.md) the `path` parameter. The array is in longest-to-shortest order. Useful when building custom [Store](./tough-cookie.store.md) implementations.
+
+**Signature:**
+
+```typescript
+export declare function permutePath(path: string): string[];
+```
+
+## Parameters
+
+<table><thead><tr><th>
+
+Parameter
+
+
+</th><th>
+
+Type
+
+
+</th><th>
+
+Description
+
+
+</th></tr></thead>
+<tbody><tr><td>
+
+path
+
+
+</td><td>
+
+string
+
+
+</td><td>
+
+the path to generate permutations for
+
+
+</td></tr>
+</tbody></table>
+**Returns:**
+
+string\[\]
```

Comment:
This doesn't seem to be formatted correctly. 🫤

Reply:
I noticed this too and will file a bug with `@microsoft/api-documenter` if one doesn't already exist.